src/dialog.html

```diff
@@ -44,6 +44,11 @@
 			transform: rotateX(-90deg);
 		}
 
+		*[data-clickable="false"]{
+		    opacity: 0.3;
+		    cursor: not-allowed !important;
```

Comment:
Yeah, I wasn't happy with this either. It was a placeholder that made it further than it should have. Will look into why the style won't apply without it.